### DISCUSSION TITLE
Regarding Next.js Caching upon Database queries

### DISCUSSION THREAD

**Discussion by SumitKumar-17** [vercel/next.js#71013] [Help] [answered]
### Summary

Hi
i am using Next.js with Prisma , i am currently making a admin portal page.

So I have to fetch the data from the database and then display on the as tables.
My problem is whenever I am performing some operations on the database, the database is updating correctly as usual ,bu the  next.js Page is not refreshing as per the Database. 

Can you help me out in this.

**Comment by suprunchuk** [comment]
Let me suggest that you use [SWR ](https://swr.vercel.app/examples/basic) if I understood your problem correctly.

**Comment by SumitKumar-17** [comment]
Okay i will look into it and try that .

**Comment by SumitKumar-17** [comment, marked as answer]
Hi 
If someone encounters this problem again, I have found  a solution:  
Add the line `const revalidate = 0;` at the top of your page component. This will disable Next.js caching for that component.
